packages/ods/src/components/spinner/documentation/style-customization-spinner.mdx

```diff
@@ -0,0 +1,12 @@
+You can add your own style on the spinner element using the part `spinner`.
+
+Example:
+```html
+<style>
```

Comment:
also would be nice to have the same order between the style tag and the component